hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/testing.md

```diff
@@ -854,7 +854,7 @@ and, for test runs in or near the S3/DDB stores, throttling events.
 
 If you want to manage capacity, use `s3guard set-capacity` to increase it
 (performance) or decrease it (costs).
-For remote `hadoop-aws` test runs, the read/write capacities of "10" each should suffice;
+For remote `hadoop-aws` test runs, the read/write capacities of "1" each should suffice;
```

Comment:
Maybe review this whole paragraph to say "the default is on-demand with read/write capacities of 0" and not worry about the mention of autoscaling